certbot/tests/cli_test.py

```diff
@@ -578,7 +578,7 @@ def test_certonly_new_request_failure(self):
     def _test_renewal_common(self, due_for_renewal, extra_args, log_out=None,
                              args=None, should_renew=True, error_expected=False):
         # pylint: disable=too-many-locals,too-many-arguments
-        cert_path = 'certbot/tests/testdata/cert.pem'
+        cert_path = test_util.vector_path('cert.pem')
```

Comment:
I believe it refers to [test vector](https://en.wikipedia.org/wiki/Test_vector).